src/components/sbb-tab-amount/sbb-tab-amount.scss

```diff
@@ -0,0 +1,15 @@
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include host-component-properties;
+
+:host {
+  display: inline-block;
+}
+
+span {
+  @include text;
+
+  vertical-align: middle;
+  line-height: 0;
+  padding-inline-start: var(--sbb-spacing-fixed-1x);
```

Comment:
True, I fixed it

Reply:
Note that I moved the padding back to `sbb-tab-amount`, as it is easier to handle when not rendered (otherwise the title would have kept the padding to its right, adding additional white space).

Reply:
Is it not possible to use the "gap" property on `sbb-tab-title`?

Reply:
Yes, I did it 👍